File: src/main/java/me/lulu/hypixelutilities/ChatListener.java
package me.lulu.hypixelutilities;

import me.lulu.hypixelutilities.util.McUtil;
import net.minecraft.util.ChatComponentText;
import net.minecraftforge.event.CommandEvent;
import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;

public class ChatListener {

    @SubscribeEvent
    public void onChat(CommandEvent e) {
        McUtil.getPlayer().addChatMessage(new ChatComponentText(e.command.getCommandName() + " " + String.join(" ", e.parameters)));
    }

}

File: src/main/java/me/lulu/hypixelutilities/ChatListener.kt
package me.lulu.hypixelutilities

import me.lulu.hypixelutilities.util.clientPlayer
import net.minecraft.util.ChatComponentText
import net.minecraftforge.event.CommandEvent
import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
import java.lang.String

class ChatListener {
    @SubscribeEvent
    fun onChat(e: CommandEvent) {
        clientPlayer.addChatMessage(ChatComponentText(e.command.commandName + " " + String.join(" ", *e.parameters)))
    }
}
